### PR TITLE
Remove ca-bundle from the flow

### PR DESCRIPTION
**Pull request by tutelaris**
### Description
In order to fix the problem with ssl certificates, it was decided to remove it completely and used a built-in certificate

This PR is dedicated to removing `ca-bundle.crt`

### CODE REVIEW

**Review by 1stvamp — DISMISSED** [review]
LGTM.

One other thing we could do is add a `ca_path` option to our wrapper and pass it through to `net.ca_path`, that would allow customers with ancient systems to supply to own CA store, or update the CA store independently of the system one.

Kind of out of scope for the fix, 99.9% of anyone these days should be covered by the default CA stores. :+1:

**Review by 1stvamp — DISMISSED** [review]

File: .circleci/config.yml [hunk 1]
```diff
@@ -66,4 +66,4 @@ workflows:
       - build: { name: run-tests-ruby-2.4, executor: ruby-24 }
       - build: { name: run-tests-ruby-2.5, executor: ruby-25 }
       - build: { name: run-tests-ruby-2.6, executor: ruby-26 }
-      - build: { name: run-tests-jruby-92, executor: jruby-92 }
+      - build: { name: run-tests-jruby-93, executor: jruby-93 }
```
[the remark this answers is not in the document]
Comment (tutelaris) [reply .circleci/config.yml:69]: Yeah, makes sense!

[the remark this answers is not in the document]
Comment (tutelaris) [reply .circleci/config.yml:69]: done 🏁

**Review by 1stvamp — DISMISSED** [review]

File: .circleci/config.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (tutelaris) [reply .circleci/config.yml:75]: Returned back ruby 2.4 and ruby 2.5 as well as jruby 9.2

**Review by 1stvamp — APPROVED** [review]

**Review by acrewdson — APPROVED** [review]

**Pull request merged**